src/tools/ResultsComparer/CommandLineOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using CommandLine;
+using CommandLine.Text;
+
+namespace ResultsComparer
+{
+    public class CommandLineOptions
```

Comment:
After looking at the code/usage of the tool, I will suggest grouping arguments under verbs: compare / analyze? With the current command line arguments usage you can use either merge or base/diff.

Reply:
I did remove the `--merged` option. So we don't need verbs anymore